packages/@o3r/core/schematics/ng-add/project-setup/index.ts

```diff
@@ -40,54 +41,69 @@ export const prepareProject = (options: NgAddSchematicsSchema) => async (tree: T
     readPackageJson, removePackages, renamedPackagesV7toV8, updateImports
   } = await import('@o3r/schematics');
   const installOtterLinter = await shouldOtterLinterBeInstalled(context);
-  const workspaceProject = tree.exists('angular.json') ? getProjectFromTree(tree) : undefined;
-  const projectType = workspaceProject?.projectType || 'application';
+  const workspaceProject = tree.exists('angular.json') ? getProjectFromTree(tree, options.projectName) : undefined;
+  const projectType = workspaceProject?.projectType;
   const internalPackagesToInstallWithNgAdd = Array.from(new Set([
     ...(projectType === 'application' ? ['@o3r/application'] : []),
-    ...(options.enableStyling ? ['@o3r/styling'] : []),
+    ...(options.enableApisManager && !!projectType ? ['@o3r/apis-manager'] : []),
```

Comment:
All these options could have been removed actually

Reply:
don't you want to wait and remove them only when we implement the dynamic presets ?